src/System.Text.Encoding.CodePages/src/System/Text/DBCSCodePageEncoding.cs

```diff
@@ -79,7 +79,9 @@ internal DBCSCodePageEncoding(int codePage, int dataCodePage, EncoderFallback en
         [System.Security.SecurityCritical]  // auto-generated
         protected override unsafe void LoadManagedCodePage()
         {
-            fixed (byte* pBytes = m_codePageHeader)
+            Debug.Assert(m_codePageHeader?.Length > 0);
```

Comment:
It's an assert, so it should be that it couldn't possibly be null or empty. It's reasonable to have an assert failure in the (deemed impossible) case of it being null rather than an NRE from within it just as there would be for the (deemed impossible) case of it being empty.